Ginger/GingerCoreCommon/Repository/BusinessFlowLib/BusinessFlow.cs

```diff
@@ -86,6 +86,20 @@ public enum eSource
             Selenium,   // From Selenium Import
             Gherkin     // From Gherking Feature file
         }
+        
+        bool mSelected;
```

Comment:
we don't need that 'Selected' anymore, no?

Reply:
yes we need it so kept